README.adoc

```diff
@@ -302,9 +274,9 @@ Results :
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
 ----
 
-To see if the tests detect a failure, remove the reset retries method on
+To see if the tests detect a failure, remove the change system property utility call on
 the `FaultToleranceTest.java` file. Rerun the Maven build. You see a test failure occur for the
-`testRetryGettingSystemProperties()` test case.
+`testFallbackForGet()` test case.
```

Comment:
Could we, at this point, have the user remove or comment out the `@Fallback` annotation in the `InventoryManager`?

I.e. introduce a bug into the application and show that the tests pick it up?

Reply:
Ah ok, let me test that out first.

Reply:
When commenting out the `@Fallback` annotation in the `InventoryManager` class, the test case failed with this error: 
```
testFallbackForGet(it.io.openliberty.guides.faulttolerance.FaultToleranceTest)  Time elapsed: 3.204 sec  <<< FAILURE!
java.lang.AssertionError: Incorrect response code from http://localhost:9080/ expected:<200> but was:<500>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at it.io.openliberty.guides.faulttolerance.FaultToleranceTest.assertResponse(FaultToleranceTest.java:85)
	at it.io.openliberty.guides.faulttolerance.FaultToleranceTest.testFallbackForGet(FaultToleranceTest.java:70)
```
This make sense, as the test case sets the config property to true, the application throws the IOException (by design and now there's no fallback method to handle it) and making the server unavailable, thus the 500 error code is seen. 

I'm not sure whether the test best picks up this failure with the display message. Can look into it further to see if there's anything we can improve. 

However, I'll double check with my team to see if this is approach we want to go with, as at this point of the guide, we're not to suggest ways to break the application and have the test to handle it. It's meant to hint the user to play further with the app/test themselves, in anyway they want, based on what they have learned in the guide.

Reply:
@Azquelt The team decided to take further look at this request at another time. A new issue is opened for it. Thank you.